tools/markdown-checker/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "markdownChecker",
```

Comment:
Is this intended to be published? If so, it should start at 0.0.0 until it’s ready and should not have a lockfile; if not, it should have `private: true`

Reply:
We didn't talk yet it should be private or not. I think the first versions can be private at the beginning. In both ways, it would be good to start at 0.0.0 in my opinion because it is not ready yet.

Reply:
If it’s intended to be published, then please remove and disable the lockfile.